pinot-tools/src/main/java/org/apache/pinot/tools/BootstrapTableTool.java

```diff
@@ -6,9 +6,9 @@
  * to you under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p>
```

Comment:
intellij decided to change this (and other whitespace changed below) with `Reformat Code`

Reply:
Can you please re-apply the [Pinot Style](https://docs.pinot.apache.org/developers/developers-and-contributors/code-setup?q=code+s#setup-ide) and reformat again? The changes in this PR doesn't follow the format